test/mock/mock-lazyloaded-file.js

```diff
@@ -0,0 +1 @@
+window.lazyLoadingWorking = true;
```

Comment:
This effects global state, so once you set it to `true` one time it will remain `true` in future tests even if lazy loading does not work. You'll need to reset it back to `false` in `beforeEach`